src/types/webhooks/modifyWebhook.ts

```diff
@@ -6,4 +6,5 @@ export interface ModifyWebhook {
   avatar?: string | null;
   /** The new channel id this webhook should be moved to */
   channelId?: string;
+  reason?: string;
```

Comment:
add a /** */ also what about the function itself?

Reply:
The function should work?
![image](https://user-images.githubusercontent.com/87189679/151145205-f86afcc4-972f-4ecf-b5a1-2f4b7a25b633.png)